routing/turns_tts_text.cpp

```diff
@@ -62,11 +67,31 @@ string GetTtsText::GetTurnNotification(Notification const & notification) const
   if (notification.m_useThenInsteadOfDistance && notification.m_turnDir == CarDirection::None)
     return string();
 
-  string const dirStr = GetTextById(GetDirectionTextId(notification));
   if (dirStr.empty())
     return string();
 
   string const distStr = GetTextById(GetDistanceTextId(notification));
+
+  if (notification.m_distanceUnits > 0 && !notification.m_nextStreet.empty()) {
+    // We're going to pronounce the street name.
+
+    // First, get rid of unpronounceable symbols.
+    std::string streetOut = notification.m_nextStreet;
+    // Semicolons are between destinations
+    // and pronounced more like commas.
+    boost::replace_all(streetOut, ";", ",");
```

Comment:
I think what @biodranik meant is this:
```cpp
std::replace( streetOut.begin(), streetOut.end(), ';', ',');
``` is doing the same thing as boost's `replace_all` in this case, but it doesn't need the huge boost dependency.